tensorboard/webapp/app_routing/store_only_utils.ts

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {parseCompareExperimentStr} from './internal_utils';
+import {CompareRouteParams} from './types';
+
+// Ideally, we would not be exposing this utility method this, but there is no
+// way to share the structured information to other stores without changing
```

Comment:
I'm under the assumption that we don't want any "other store" to use this fn to parse route params, only the Experiment reducer, specifically?

If so, could we make this comment explicit about that, something like "we want the Experiment reducer to be able to listen to `navigated` actions and extract the alias from the RouteParams in the action payload, so that the experiment store can save aliases in its state".

If the intent is actually to let any other store use this, then disregard this comment.

Reply:
Left some comment and decided to use visibility to actually enforce this instead.